core/src/main/java/edu/wpi/grip/core/events/SourceStartedEvent.java

```diff
@@ -0,0 +1,19 @@
+package edu.wpi.grip.core.events;
+
+
+import edu.wpi.grip.core.Source;
+
+/**
+ * An event that occurs when a source is started.
```

Comment:
When the user requests a source is started, or when a source informs the user it has started?  It would be helpful to explain which one in the documentation for this event.